tests/docker/config.json

```diff
@@ -0,0 +1,86 @@
+{
```

Comment:
this file doesn't seem correct at all...

Reply:
This file is not used or needed.  It has been removed.